docs/ispc.rst

```diff
@@ -4252,6 +4252,18 @@ use Newton-Raphson.
     float rcp_fast(float v)
     uniform float rcp_fast(uniform float v)
 
+The ``fmodf()`` functions compute the floating-point remainder of the division
+operation x/y (its implementation is based on the ``rcp`` and ``trunc`` functions):
```

Comment:
Do we want implementation details note in the docs?

I would rather point out that it's semantics is equivalent to C/C++ lib functions.

Reply:
Fixed